mapfishapp/src/main/webapp/app/js/GEOR_mapinit.js

```diff
@@ -313,11 +313,23 @@ GEOR.mapinit = (function() {
         var records = [], record;
         var errors = [], count = 0;
         Ext.each(initState, function(item) {
-            if (item.type == "WMSLayer" || item.type == "WFSLayer") {
+            if ( (item.type == "WMSLayer" || item.type == "WFSLayer") && item.type == type+'Layer' ) {
```

Comment:
OK, sorry, I did no see "When I tested it, I can't load WMS and WFS servers simultaneously, so I added a condition over the layer Type in updateStoreFromWxSLayer()."